lldb/source/Symbol/ObjectFile.cpp

```diff
@@ -747,3 +747,8 @@ void llvm::format_provider<ObjectFile::Strata>::format(
     break;
   }
 }
+
+llvm::StringRef ObjectFile::GetReflectionSectionIdentifier(
+    swift::ReflectionSectionType section) {
+  return "";
```

Comment:
should this assert instead? It's always supposed to be overridden, no?

Reply:
@dcci I think there are some subclasses where it doesn't make sense to implement this (`ObjectFileJIT`, `YAMLObjectFile`). But then, calling this method from any of those classes would probably indicate something has gone wrong, so I agree an assertion is in order.